node/get_status_node.go

```diff
@@ -120,6 +119,7 @@ func New(transactor *transactions.Transactor, gethAccountManager *accsmanagement
 		gethAccountManager: gethAccountManager,
 		logger:             logger,
 		publicMethods:      make(map[string]bool),
+		accountsPublisher:  pubsub.NewPublisher(),
```

Comment:
@dlipicar if this publisher is created here and shared among different services, doesn't this one need to be responsible for closing it instead of doing that from the accounts service?